### PR TITLE
feat: #3002 change spec and migrate model

### PR DESCRIPTION
**Pull request by Chlenix**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ COPY spec ./spec
 COPY config ./config
 COPY test ./test
 COPY src ./src
+COPY migrations ./migrations
```
Comment (azaxarov) [inline Dockerfile]: No need to copy this to build container

File: Dockerfile [hunk 2]
```diff
@@ -29,5 +30,6 @@ COPY --from=build /usr/src/app/config ./config
 COPY --from=build /usr/src/app/spec ./spec
 COPY --from=build /usr/src/app/dist/src ./dist/src
 COPY --from=build /usr/src/app/node_modules ./node_modules
+COPY --from=build /usr/src/app/migrations ./migrations
```
Comment (azaxarov) [inline Dockerfile]: use `COPY migrations ./migrations` directly here

File: migrate-mongo-config.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+// In this file you can configure migrate-mongo
```
Comment (azaxarov) [inline migrate-mongo-config.js]: Remove comments

File: src/rabbitmq/rabbitmq.ts [hunk 4]
```diff
@@ -81,17 +80,21 @@ export const rabbitConnect = (): void => {
 };
 
 export const publishDataSource = ({
-  publisherId = ''
+  id = 0,
```
Comment (azaxarov) [inline src/rabbitmq/rabbitmq.ts]: why 0?

Comment (Chlenix) [reply src/rabbitmq/rabbitmq.ts]: What do you suggest?

File: migrate-mongo-config.js (hunk 3 above)
Comment (Chlenix) [reply migrate-mongo-config.js]: They're autogenerated, why bother

File: Dockerfile [hunk 5]
```diff
@@ -24,10 +24,13 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
...
 
+COPY migrations ./migrations
+
 COPY --from=build /usr/src/app/package.json ./
 COPY --from=build /usr/src/app/config ./config
 COPY --from=build /usr/src/app/spec ./spec
 COPY --from=build /usr/src/app/dist/src ./dist/src
 COPY --from=build /usr/src/app/node_modules ./node_modules
 
+
```
Comment (azaxarov) [inline Dockerfile]: OCD hehe

**Pull request merged**